src/controllers/movie.js

```diff
@@ -0,0 +1,159 @@
+const {
+  PrismaClientKnownRequestError,
+  PrismaClientValidationError,
+} = require("@prisma/client/runtime/library");
+const {
+  getMoviesDb,
+  getMoviesWhereAndDb,
+  getMoviesWhereLtDb,
+  getMoviesWhereGtDb,
+  createMovieDb,
+  createMovieWithScreeningsDb,
+  getMovieByIdDb,
+  updateMovieDb,
+  getMovieByTitleDb,
+  updateMovieWithScreeningsDb,
+} = require("../domains/movie");
+
+const getMovies = async (req, res) => {
+  let movies;
+  const { runtimeLt, runtimeGt } = req.query;
+  if (runtimeLt && runtimeGt) {
+    const runtimeLtNum = Number(runtimeLt);
+    const runtimeGtNum = Number(runtimeGt);
+    movies = await getMoviesWhereAndDb(runtimeGtNum, runtimeLtNum);
+  }
+
+  if (runtimeLt && !runtimeGt) {
+    const runtimeLtNum = Number(runtimeLt);
+    movies = await getMoviesWhereLtDb(runtimeLtNum);
+  }
+
+  if (runtimeGt && !runtimeLt) {
+    const runtimeGtNum = Number(runtimeGt);
+    movies = await getMoviesWhereGtDb(runtimeGtNum);
+  }
+
+  if (!runtimeLt && !runtimeGt) {
+    movies = await getMoviesDb();
+  }
```

Comment:
This here looks like a query building situation. I think this would be better if you moved it all into its own function that takes care of building queries in a specific way to de-clutter your controller